cli/cli/cli.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Command, Inject, Injectable } from "../deps.ts";
+
+@Injectable({ global: false })
+export class CliService {
+  constructor(
+    @Inject("MVF_COMMANDS") private readonly commands: Command[],
+  ) {
+  }
+
+  startProgram() {
+    const program = new Command("mvf");
+
+    program.version("0.0.1");
```

Comment:
Since we have an installer script now. The install script can note the version its pulling and write it in the .mv folder. I don't like hard coding the version in the cli.